spec/features/beis_users_can_create_a_programme_level_activity_spec.rb

```diff
@@ -69,7 +69,7 @@
       )
       expect(created_activity.commitment.value).to eq(activity.commitment.value)
 
-      expect(created_activity.publish_to_iati).to eq(true)
```

Comment:
Ideally in future, I think it'd be good for refactors like this that are kind of separate from the main feature and touch the whole codebase to be part of their own PR, as much of a faff as that is. This'd make it quicker to review an individual PR, easier to track down any merges or PRs that might have introduced issues (even though a lot of that _is_ possible with commits)

Reply:
Yeah, this ballooned a little. Originally I'd only changed three `eql` in one file to `eq`, but then I decided to do it all when changing to `be`. Should've separated it out!